jdaviz/components/plugin_add_results.vue

```diff
@@ -22,7 +22,15 @@
     ></plugin-viewer-select>
 
     <v-row v-else>
-      <v-switch
+      <v-switch v-if="label_overwrite"
+        class="hide-input"
+        :label="'Show in '+add_to_viewer_items[1].label"
+        :disabled="true"
```

Comment:
I find this disabled GUI confusing. Looks like you don't even need a switch anymore? Maybe I am missing something after being away for a week.

Reply:
I tried mimicking the same layout without using a switch (the idea being to avoid a jarring shift in layout or styling), but ultimately found this to be simpler to implement.  Other ideas welcomed!

Reply:
Removing the switch entirely could also be a good temporary solution. I don't have strong feelings either way if we want to leave this up to the POs.

Reply:
I didn't see @kecnry's comment when I left my comment. Since this should hopefully be temporary, I'm ok with proceeding with this PR as is.